specification/storage/resource-manager/Microsoft.Storage/stable/2021-02-01/storage.json

```diff
@@ -0,0 +1,4468 @@
+        "message": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets an error message explaining the Reason value in more detail."
+        }
+      },
+      "description": "The CheckNameAvailability operation response."
+    },
+    "SkuInformation": {
+      "properties": {
+        "name": {
+          "$ref": "./common.json#/definitions/SkuName"
+        },
+        "tier": {
+          "$ref": "./common.json#/definitions/Tier"
+        },
+        "resourceType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the resource, usually it is 'storageAccounts'."
+        },
+        "kind": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Indicates the type of storage account.",
+          "enum": [
+            "Storage",
+            "StorageV2",
+            "BlobStorage",
+            "FileStorage",
+            "BlockBlobStorage"
+          ],
+          "x-ms-enum": {
+            "name": "Kind",
+            "modelAsString": true
+          }
+        },
+        "locations": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The set of locations that the SKU is available. This will be supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.)."
+        },
+        "capabilities": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SKUCapability"
+          },
+          "description": "The capability information in the specified SKU, including file encryption, network ACLs, change notification, etc."
+        },
+        "restrictions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Restriction"
+          },
+          "description": "The restrictions because of which SKU cannot be used. This is empty if there are no restrictions."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "Storage SKU and its properties"
+    },
+    "CustomDomain": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the custom domain name assigned to the storage account. Name is the CNAME source."
+        },
+        "useSubDomainName": {
+          "type": "boolean",
+          "description": "Indicates whether indirect CName validation is enabled. Default value is false. This should only be set on updates."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "The custom domain assigned to this storage account. This can be set via Update."
+    },
+    "SasPolicy": {
+      "properties": {
+        "sasExpirationPeriod": {
+          "type": "string",
+          "example": "1.15:59:59",
+          "description": "The SAS expiration period, DD.HH:MM:SS."
```

Comment:
@hyhdtc could you add examples for your new features including SasPolicy and KeyPolicy usage?

Reply:
Added samples in StorageAccount Create/Update/GetProperties